Resources/Prototypes/Objectives/objectiveGroups.yml

```
@@ -33,8 +33,9 @@
 - type: weightedRandom
   id: TraitorObjectiveGroupState
   weights:
-    EscapeShuttleObjective: 1
+    EscapeShuttleObjective: 0.4 #Harmony change
```

I still want the old value to be stored for cross referencing
```suggestion
    EscapeShuttleObjective: 0.4 #Harmony change: 1 -> 0.4
```

---

I see what you mean now, should be fixed.